aptos-move/aptos-gas/src/transaction.rs

```diff
@@ -33,7 +33,7 @@ crate::params::define_gas_parameters!(
         [
             intrinsic_gas_per_byte: InternalGasPerByte,
             "intrinsic_gas_per_byte",
-            8
+            2_000
```

Comment:
Why is this changed to 2000 now?

Reply:
This is to account for things like deserialization and bytecode verification